AzureCalling/Controllers/CallViewController.swift

```diff
@@ -236,7 +236,17 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
     @IBAction func onToggleMute(_ sender: UIButton) {
         sender.isSelected = !sender.isSelected
-        sender.isSelected ? callingContext.mute { _ in} : callingContext.unmute { _ in}
+        (sender.isSelected ? callingContext.mute : callingContext.unmute) { [weak self] result in
+            guard let self = self else {
+                return
+            }
+            if case .success(let call) = result,
+               let isMuted = call?.isMuted {
```

Comment:
We shouldn't need the `Call` object here. We should be able to update the mute indicator based on success/error.

Reply:
Made use of `sender.isSelected` instead